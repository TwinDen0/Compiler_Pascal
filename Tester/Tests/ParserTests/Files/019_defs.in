program CheckDefs;
var
a,b : integer = 5;
begin
;
end.