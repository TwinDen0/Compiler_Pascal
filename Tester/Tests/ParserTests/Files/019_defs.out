(3,15) ERROR: Only one variable can be initialized
